packages/tools/src/enums/Events.ts

```diff
@@ -96,6 +96,30 @@ enum Events {
    */
   ANNOTATION_RENDERED = 'CORNERSTONE_TOOLS_ANNOTATION_RENDERED',
 
+  /**
+   * Triggers on the eventTarget when an annotation label added or updated.
+   * Make use of {@link EventTypes.AnnotationLabelChangeEventType | Annotation Label Change Event Type}
+   * for typing your event listeners for this annotation label change event, and see what
+   * event detail is included in {@link EventTypes.AnnotationLabelChangeEventDetail | Annotation Label Change Event Detail}.
+   */
+  ANNOTATION_LABEL_CHANGE = 'CORNERSTONE_TOOLS_ANNOTATION_LABEL_CHANGE',
+
+  /**
+   * Triggers on the eventTarget when an annotation interpolation process completed.
+   * Make use of {@link EventTypes.AnnotationInterpolationCompletedEventType | Annotation Interpolation process Completed Event Type}
+   * for typing your event listeners for this annotation interpolation complete event, and see what
+   * event detail is included in {@link EventTypes.AnnotationInterpolationCompletedEventDetail | Annotation interpolation process Event Detail}.
+   */
+  ANNOTATION_INTERPOLATION_PROCESS_COMPLETED = 'CORNERSTONE_TOOLS_ANNOTATION_INTERPOLATION_PROCESS_COMPLETED',
```

Comment:
Could this be named something generic, so the same event could be used for other long-running annotation processing - maybe just:
ANNOTATION_PROCESSING_COMPLETED
Or maybe even have a few of them, so you can show progress as well as completed, but all against a single event, eg
ANNOTATION_PROCESSING
and then allow for a progress/status update to occur.
That is such a common pattern that it would be really nice to support it generically.
@sedghi - how do you think this should be done, as a final event or a full progress, and is it ok to use for other annotation processing types?